RoaringBitmap/src/main/java/org/roaringbitmap/RoaringBitmap.java

```diff
@@ -551,6 +551,15 @@ public static RoaringBitmap bitmapOfUnordered(final int... data) {
     return writer.getUnderlying();
   }
 
+  /**
+   * @see #add(long, long)
+   */
+  public static RoaringBitmap bitmapOfRange(long min, long max) {
```

Comment:
I guess that's fine though, of course, it is not a terribly powerful function. :-) 

(This is not a request for change.)

Reply:
Is part of the comment missing? Do you want a test to cover this directly?

Reply:
No. That's my full comment. I guess that my point is that this function is not essential. But then we have many such small functions already.